libs/mocks/WordPressMocks/src/main/assets/mocks/mappings/wpcom/reader/rest_v12_read_following_mine.json

```diff
@@ -1,7 +1,7 @@
 {
     "request": {
         "method": "GET",
-        "urlPattern": "/rest/v1.2/read/following/mine.*"
+        "urlPattern": "/rest/v1.(1|2)/read/following/mine.*"
```

Comment:
Is it ok to update this mock file like this? [`rest_v11_me_sites.json`](https://github.com/wordpress-mobile/WordPress-Android/blob/develop/libs/mocks/WordPressMocks/src/main/assets/mocks/mappings/wpcom/me/rest_v11_me_sites.json) seems to do the same, but I'm not sure what's the best practice here – specially because these files are prefixed with the API version.

Reply:
Yep looks good